iBurn/src/main/java/com/gaiagps/iburn/MapboxMapFragment.kt

```diff
@@ -902,7 +914,7 @@ class MapboxMapFragment : Fragment() {
                         mappedItems.add(userPoi)
                         if (marker != null) {
                             mappedCustomMarkerIds[marker.id] = userPoi
-                            callback?.invoke(marker)
+                            // callback?.invoke(marker)TODO
```

Comment:
What are the repercussions of commenting this out?

Reply:
Addressed here: https://github.com/iBurnApp/iBurn-Android/pull/13/commits/2c17d4bf1f02c514408d8a2d04f5cd8ff19ce362